src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -3,103 +3,309 @@
-        - name: Install the ACI-Connector from a specific custom chart.
+        - name: Install the ACI Connector from a custom Helm chart.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --chart-url <CustomURL>
-"""
+""".format(sp_cache=SERVICE_PRINCIPAL_CACHE)
+
 helps['aks remove-connector'] = """
     type: command
-    short-summary: Undeploy the ACI-Connector from an AKS cluster.
+    short-summary: Remove the ACI Connector from a managed Kubernetes cluster.
+    long-summary: |-
+        Stops the cluster from deploying Azure Container Instances.
+        See https://github.com/Azure/aci-connector-k8s for more details.
+    parameters:
+        - name: --connector-name
+          type: string
+          short-summary: Name of the ACI Connector.
+        - name: --graceful
+          type: bool
+          short-summary: Use a "cordon and drain" strategy to evict pods safely before removing the ACI node.
+        - name: --os-type
+          type: string
+          short-summary: Remove support for deploying ACIs of this operating system type.
     examples:
-        - name: Undeploy the ACI-Connector on an AKS cluster.
+        - name: Remove the ACI Connector from a managed Kubernetes cluster.
           text: az aks remove-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector
-        - name: Undeploy the ACI-Connector on an AKS cluster using the graceful mode
+        - name: Remove the ACI Connector from a cluster using the graceful mode.
           text: az aks remove-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --graceful
 """
+
+helps['aks scale'] = """
+    type: command
+    short-sumnary: Scale the node pool in a managed Kubernetes cluster.
+    parameters:
+        - name: --node-count -c
+          type: int
+          short-summary: Number of nodes in the Kubernetes node pool.
+"""
+
+helps['aks show'] = """
+    type: command
+    short-summary: Show details of a managed Kubernetes cluster.
+"""
+
+helps['aks upgrade'] = """
+    type: command
+    short-summary: Upgrade a managed Kubernetes cluster to a newer version.
+    long-summary: "NOTE: Kubernetes may be unavailable during cluster upgrades."
+    parameters:
+        - name: --kubernetes-version -k
+          type: string
+          short-summary: Version of Kubernetes to upgrade the cluster to, such as "1.7.7" or "1.8.2".
+          populator-commands:
+          - "`az aks get-versions`"
+"""
+
 helps['aks wait'] = """
     type: command
-    short-summary: Wait for a Kubernetes cluster to reach a desired state.
+    short-summary: Wait for a managed Kubernetes cluster to reach a desired state.
+    long-summary: If an operation on a cluster was interrupted or was started with `--no-wait`, use this command to
+                  wait for it to complete.
+    examples:
```

Comment:
The first three examples can be removed since they only show simple usage of the command.  If waiting for a specific condition is tied to other `aks` subcommands, that should be mentioned in the documentation for each argument.